config/web-ldap-admin.php

```diff
@@ -49,10 +49,14 @@
     'mostrarFoto' => env('WSFOTO', 0),
 
     # Forçar trocar senha no pŕoximo login do windows
-    # se o login com AD é usado em outros sistemas, como aplicações web 
+    # se o login com AD é usado em outros sistemas, como aplicações web
     # ou em terminais com linux deixar essa opção como 0 (falsa)
     # pois o usuário fica travado e não consegue logar em nada a não ser nos windows.
     # Por default está 1 (true) pois assim estava no Ldap/User.php
     # Válido para criação de contassincronizadas
     'obrigaTrocarSenhaNoWindows' => env('OBRIGA_TROCAR_SENHA_NO_WINDOWS', 1),
+
+    # Curso de graduação x Habilitação x Setor (Departamento de ensino)
+    # Se não estiver configurada no env o método setorAluno será utilizado
+    'curHabSet' => (env('CUR_HAB_SET')) ? array_values(json_decode(file_get_contents('/app/' . env('CUR_HAB_SET')), true)) : [],
```

Comment:
* talvez a variável possa chamar grCursoSetor pois pode ser que tenha no futuro pode ter pgCursoSetor;
* o env acho que usa o root do filesystem como referência. Então seria melhor usar um caminho relativo no .env (./storage/app/xxx.json) e não concatenar aqui. Como o file_get_contents aceita url, essa mudança sem concatenar permitiria armazenar o json em uma url compartilhada.

* Se o formato do json for conforme abaixo, acho que não precisaria do array_values.
```
[
      {
        "codcur": "27223",
        "codhab": "101",
        "codset": "1231"
      },
      {
        "codcur": "27302",
        "codhab": "101",
        "codset": "1232"
      }
]
```

Reply:
Ok quanto ao caminho relativo, mas porque uma url compartilhada?

Reply:
Se precisa de array_values... só testando, quando fiz foi no objetivo converter em array.

Reply:
> Se precisa de array_values... só testando, quando fiz foi no objetivo converter em array.

No exemplo que postei é a forma correta de repesentar array no json com indice numérico.

Reply:
Postou onde?

Reply:
Da forma que vc formatou inciando e terminando com colchetes não funcionou. Deixei com as chaves mesmo.

Reply:
Agora acertei a formatação como você disse